Add cleaning cluster nodes

superseb opened the pull request:
https://github.com/rancher/docs/issues/75

galal-hussein commented on content/rancher/v2.x/en/installation/removing-rancher/cleaning-cluster-nodes/_index.md:
````diff
@@ -0,0 +1,145 @@
+```
+rm -rf /etc/ceph \
+       /etc/cni \
+       /etc/kubernetes \
````
/etc/kubernetes
/var/lib/etcd/
/etc/cni/
/opt/cni/
/var/lib/cni/
/var/run/calico/
/etc/kubernetes/.tmp/

all of these previous directories are being cleaned by `rke remove` there is no need to include them

galal-hussein commented on content/rancher/v2.x/en/installation/removing-rancher/cleaning-cluster-nodes/_index.md:
```diff
@@ -0,0 +1,145 @@
+weight: 375
+---
+
+When adding a node to a cluster, the following items are created/configured on the node:
```
i think its important to mention that when **removing a cluster** rke will clean each host if it can connect to it mentioning the what rke does on cleaning including cleaning directories and k8s components

galal-hussein commented on content/rancher/v2.x/en/installation/removing-rancher/cleaning-cluster-nodes/_index.md:
```diff
@@ -0,0 +1,145 @@
+
+>**Warning:** The commands listed below may break default node behavior. The recommended way is to restart the node to restore the original state of the iptables rules.
+
+* How to remove all iptables rules
```
these are really dangerous set of commands and it can block the user from the machine if applied in a wrong way, and will break docker as far as i can see, because you are removing all the nat rules including docker's nat rules
so its best to just restart the machine

galal-hussein replied:
i don't think we should mention these rules even with a warning

superseb commented:
- Added two scenarios, removing node when active (automatic cleaning) or manual. Details need to be included for manual, as you can't rely on the fact that the node is always reachable.
- Removed iptables commands to remove/flush

superseb commented:
@deniseschannon ^^
